packages/@okta/vuepress-site/docs/guides/submit-app/main/scim/submit.md

```diff
@@ -1,7 +1,7 @@
 
 * **What SCIM version should this integration use?**: Choose either **SCIM 2.0** (Okta preferred version) or **SCIM 1.1**.
 
-* **Link to configuration guide**: Your configuration guide (in either HTML or PDF format) should have step-by-step instructions on how to set up provisioning between Okta and your systems. See [Prepare a customer-facing configuration guide](/docs/guides/submit-app/scim/main/#prepare-a-customer-facing-configuration-guide).
+* **Link to configuration guide**: Your configuration guide (in HTML or PDF format) should have step-by-step instructions on how to set up provisioning between Okta and your systems. See [Customer configuration document guidelines](/docs/guides/submit-app-prereq/main/#customer-configuration-document-guidelines).
```

Comment:
This also applies to the SAML and OIDC docs too.

Reply:
Sounds fair. I'll update the doc guidelines to remove the "whatever format works best" line. But I'll keep HTML and PDF here since this isn't something that the Ops team approve that we change. I think the Google docs format is still ok since it's a URL link (in HTML), but needs to be publicly accessible when the integration is live in the catalog.